src/Symfony/Component/Security/Csrf/TokenStorage/SessionTokenStorage.php

```diff
@@ -26,17 +27,24 @@ class SessionTokenStorage implements ClearableTokenStorageInterface
      */
     public const SESSION_NAMESPACE = '_csrf';
 
-    private $session;
+    private $requestStack;
+    private $session; // deprecated, to be removed in Symfony 6.0
```

Comment:
we'd better remove the property and always use requestStack imho

Reply:
that would be a BC break. Unless I build a dummy requestStack :-/

Reply:
updated to use a dummy RequestStack. Please have a look to the diff